module/input/SensorFilter/data/config/SensorFilter.yaml

```diff
@@ -42,3 +42,10 @@ motion_filter:
       rotation: [0.1, 0.1, 0.1, 0.1] # Not very sure of rotation (QX, QY, QZ, QW)
       rotational_velocity: [0.1, 0.1, 0.1] # Fairly unsure about our rotational velocity
       gyroscope_bias: [1, 1, 1] # Very unsure about our gyroscope bias
+
+mahony:
+  Kp: 10.8279
+  Ki: 17.8421
+  ts: 0.0085
+  initial_quat: [1, 0, 0, 0]
+  bias: [0, 0, 0]
```

Comment:
what are these variables? They have no comments on what they are

Reply:
Mostly opened this PR so I know what files to revert. This is from the Mahony Filter branch, so it will be removed.

Reply:
No problem, didn't realise it was a draft PR

Reply:
It's fine to review draft PRs, but this one is quite far from a review at this time 😅